File: BackEnd/src/main/java/ma/ensate/rh/ensate_rh_backend/dto/LoginResponse.java
package ma.ensate.rh.ensate_rh_backend.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class LoginResponse {
    private Long id;
    private String nom;
    private String prenom;
    private String email;
    private String username;
    private String token;
}

File: FrontEnd/app/src/main/java/com/ensa/gestionpersonnel/data/remote/dto/LoginResponse.kt
package com.ensa.gestionpersonnel.data.remote.dto

/**
 * Réponse de l'API d'authentification.
 * Adaptée au modèle métier du backend Spring Boot.
 */
data class LoginResponse(
    val id: Long,
    val nom: String,
    val prenom: String,
    val email: String,
    val username: String,
    val token: String
)
